app/assets/javascripts/controllers/container_dashboard/util/charts-mixin.js

```diff
@@ -24,9 +24,31 @@ angular.module('miq.util').factory('chartsMixin', [function chartMixinFactory ()
     }
   };
 
+  var processHeatmapData = function(data) {
+    data.sort(function(a, b) {return b.value - a.value})
+    var result = [];
+
+    for (var i = 0, l = data.length; i < l; i++) {
+      var d = data[i]
+      var percent = d.value * 100
+      var used = Math.floor(d.value * d.info.total)
+      var available = d.info.total - used
+      var tooltip =`${d.info.node} : ${d.info.provider}\n${percent}%: ${used} Used of ${d.info.total} Total\n - ${available} Available`
```

Comment:
Also, I would remove the - before the ${available} as it makes the available number look negative